completions/fish/crio.fish

```diff
@@ -131,6 +131,7 @@ complete -c crio -n '__fish_crio_no_subcommand' -f -l rdt-config-file -r -d 'Pat
 complete -c crio -n '__fish_crio_no_subcommand' -f -l read-only -d 'Setup all unprivileged containers to run as read-only. Automatically mounts the containers\' tmpfs on `/run`, `/tmp` and `/var/tmp`.'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l registry -r -d 'Registry to be prepended when pulling unqualified images. Can be specified multiple times.'
 complete -c crio -n '__fish_crio_no_subcommand' -l root -s r -r -d 'The CRI-O root directory.'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l run-as-spc-when-hostnetwork -d 'Determines whether pods are run as SPC, when hostNetwork is set.'
```

Comment:
nit: maybe `Whether SELinux should be disabled within a pod when it is running in the host network namespace`